ops/terraform/remote_files/scripts/install-node.sh

```diff
@@ -122,7 +122,7 @@ function install-bacalhau-from-source() {
   sudo apt-get -y install --no-install-recommends jq nodejs make
   git clone --branch ${BACALHAU_BRANCH} https://github.com/bacalhau-project/bacalhau.git
   pushd bacalhau
-  pushd webui && npm install && popd
+  pushd webui && yarn install && popd
```

Comment:
Has this been tested? I don't think yarn will be installed here.

Reply:
good question - yarn ships with many installs of node, but no reason not to include the install command as well.